test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -221,7 +223,7 @@ function RunSaveHelpTests
                 $saveHelpFolder = Join-Path $TestDrive (Get-Random).ToString()
                 New-Item  $saveHelpFolder -Force -ItemType Directory
 
-                It "Validate Save-Help for the '$moduleName' module" {
+                It "Validate Save-Help for the '$moduleName' module" -Pending:$Pending {
```

Comment:
@SteveL-MSFT Do you have any further concerns? If not, I'd like to merge this PR to get CI running again